go.mod

```diff
@@ -14,14 +16,14 @@ require (
 	github.com/stretchr/testify v1.10.0
 	github.com/urfave/cli/v2 v2.27.5
 	golang.org/x/sys v0.32.0
-	k8s.io/api v0.32.0
-	k8s.io/apimachinery v0.32.0
-	k8s.io/client-go v0.32.0
-	k8s.io/component-base v0.32.0
-	k8s.io/dynamic-resource-allocation v0.32.0
+	k8s.io/api v0.33.0
+	k8s.io/apimachinery v0.33.0
+	k8s.io/client-go v0.33.0
+	k8s.io/component-base v0.33.0
+	k8s.io/dynamic-resource-allocation v0.33.0
 	k8s.io/klog/v2 v2.130.1
-	k8s.io/kubelet v0.32.0
-	k8s.io/kubernetes v1.32.0
+	k8s.io/kubelet v0.33.0
+	k8s.io/kubernetes v1.33.0
 	k8s.io/mount-utils v0.32.0
```

Comment:
Question: Should we bump `mount-utils` to `v0.33.0` too?

Reply:
I didn't overdo my minimization effort, but I tried to find the "smallest" set of bumps to support the new dependency.

I read your question as if you tend to doing this bump in here, too, even though it may not be strictly required.

Reply:
Update: I added the switch to mount-utils 0.33 to this branch. The change set is small, seems to contain bug fixes for Windows.